docs/app/documentation/component-docs/localization-editor/examples/localization-editor-forms-example.component.html

```diff
@@ -0,0 +1,22 @@
+<form [formGroup]="customForm">
+    <fd-localization-editor [compact]="true">
+        <fd-localization-editor-main [label]="'EN'">
+            <input fd-localization-editor-input formControlName="en" type="text" placeholder="EN">
+        </fd-localization-editor-main>
```

Comment:
use a loop for these maybe? It's nice to show how to use the ngFor instead of hardcoded components